account_payment_order/report/account_payment_order.xml

```diff
@@ -81,7 +81,12 @@
                                 <span t-esc="get_bank_account_name(line.partner_bank_id)"/>
                             </td>
                             <td class="text-center">
-                                <span t-raw="'%s &lt;br&gt;' % line.move_line_id.invoice_id.number or line.move_line_id.move_id.name or ''"/>
+                                <t t-if="line.move_line_id.move_id and 'in_' in line.move_line_id.move_id.type">
```

Comment:
This should be cherry-picked instead of added manually to your commit

Reply:
If I cherry-pick this, I still need to edit the code...
So it will be my commit after all, not?

Reply:
Not really: you cherry-pick the commit before your migration commits, and then edit what needed in your migration commit, but not whole line. Any way, as this is a nitpicking and we want to have this merged ASAP, let's continue this time, but please take it into account next time. You are altering the commit history, stealing the credit of this to the contributor and now the reason why this has been introduced, explained in the commit message, is lost.

Reply:
![image](https://user-images.githubusercontent.com/8638295/76554611-75a76c80-6496-11ea-9257-139020d599f8.png)
PR https://github.com/OCA/bank-payment/pull/680 is already included!

Reply:
OK, that's great then! Now please check the rest of the comments.